### DISCUSSION TITLE
What is the purpose of the "Metadata" field on documents?

### DISCUSSION THREAD

**Discussion by alexmadey-oc** [doccano/doccano#1276] [Q&A]
I noticed it is a "{}" by default, which makes me think this is supposed to be a JSON. 

The reason I ask is that I am interacting with doccano through the API, and this is a field I can set when creating a document. I want to have a way to store my own ID for documents somewhere, and this field seems like a possibility. I just don't want to break anything by doing this.

**Comment by fmogetta** [comment]
Today i tried to import a labeled JSONL exported from a previous version of Doccano, and to my surprise, the "ID" of each imported document was populated as Metadata. This made me think that this metadata could, for example, be used to track the original document name, something that I´m missing in Doccano.
